### PR TITLE
More readme edits

### PR DESCRIPTION
**Pull request by tphalp**
Some more changes that came about while recording the demo.

Resolves UCP-214

### CODE REVIEW

File: apps/server/src/server.js [hunk 1]
```diff
@@ -83,15 +83,13 @@ function renderDefaultPage(req, res, html) {
...
 app.get('/', async function (req, res) {
-  info(`Serving UI resources from ${config.ResourcePrefix}`)
   req.metricsPath = '/catchall'
   const resourcePath = `${config.ResourcePrefix}${config.ResourceVersion}${req.path}`
   await _wget(resourcePath).then((html) => {
     renderDefaultPage(req, res, html)
   })
 })
 app.get('*', async function (req, res) {
-  info(`Serving UI resources from ${config.ResourcePrefix}`)
```
Comment (tphalp) [inline apps/server/src/server.js:94]: Unnecessary

File: apps/server/src/server.js [hunk 2]
```diff
@@ -83,15 +83,13 @@ function renderDefaultPage(req, res, html) {
 }
 
 app.get('/', async function (req, res) {
-  info(`Serving UI resources from ${config.ResourcePrefix}`)
```
Comment (tphalp) [inline apps/server/src/server.js:86]: Unnecessary

File: PREFERENCES.md [hunk 3]
```diff
@@ -17,9 +17,12 @@ Follow these links to sign-up for credentials for each provider that the UCW cur
 
 - MX - [Register for a developer account](https://dashboard.mx.com/sign_up)
   - Once you have created an account, go to [https://dashboard.mx.com/home](https://dashboard.mx.com/home), and look for your `API Key` and `Client ID`. Put those values in `./apps/server/.env`
+  - Note that MX requires that your IP address(es) be approved and added to their allow-list, prior to being able to connect. This can be done in their dashboard.
```
Comment (tphalp) [inline PREFERENCES.md:20]: Added note about IP allowlist.

File: PREFERENCES.md [hunk 4]
```diff
@@ -17,9 +17,12 @@ Follow these links to sign-up for credentials for each provider that the UCW cur
...
 - Sophtron - [Register for a developer account](https://sophtron.com/Account/Register)
   - Once you have created an account, go to [https://sophtron.com/Manage](https://sophtron.com/Manage), and look for your `UserId` and `AccessKey`. Put those values in `./apps/server/.env`
 
+_Note: Providers require a contract to access their production APIs._
+
```
Comment (tphalp) [inline PREFERENCES.md:25]: Another note about prod access

**Review by Craiting — APPROVED** [review]

**Pull request merged**